zookeeper-api/src/main/java/org/apache/helix/zookeeper/api/factory/RealmAwareZkClientFactory.java

```diff
@@ -45,11 +45,17 @@ RealmAwareZkClient buildZkClient(RealmAwareZkClient.RealmAwareZkConnectionConfig
    * @param metadataStoreRoutingData
    * @return RealmAwareZkClient
    */
+
   // TODO: remove MetadataStoreRoutingData
   default RealmAwareZkClient buildZkClient(
       RealmAwareZkClient.RealmAwareZkConnectionConfig connectionConfig,
       MetadataStoreRoutingData metadataStoreRoutingData) {
     return buildZkClient(connectionConfig, new RealmAwareZkClient.RealmAwareZkClientConfig(),
         metadataStoreRoutingData);
+    /*
+  default RealmAwareZkClient buildZkClient(
+      RealmAwareZkClient.RealmAwareZkConnectionConfig connectionConfig) {
+    return buildZkClient(connectionConfig, new RealmAwareZkClient.RealmAwareZkClientConfig());
+  */
```

Comment:
Why is this necessary? Since we already have a TODO, we don't need to leave this here as a block comment.

In general, our coding standards are such that we avoid code that's blocked out like this. Please consider removing this as you revise this PR :)